edge-modules/azure-monitor/src/Program.cs

```diff
@@ -0,0 +1,129 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+namespace Microsoft.Azure.Devices.Edge.Azure.Monitor
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using Microsoft.Azure.Devices.Client;
+    using Microsoft.Azure.Devices.Client.Transport.Mqtt;
+    using Microsoft.Azure.Devices.Shared;
+    using Microsoft.Extensions.Logging;
+    using Newtonsoft.Json;
+    using Microsoft.Azure.Devices.Edge.Util;
+
+
+    using System.Diagnostics;
+
+    internal static class Program
+    {
+        private static void WaitForDebugger()
+        {
+            LoggerUtil.Writer.LogInformation("Waiting for debugger to attach");
+            for(int i = 0; i < 300 && !Debugger.IsAttached; i++)
+            {
+                Thread.Sleep(100);
+            }
+            Thread.Sleep(250);
+        }
+
+        public static int Main() => MainAsync().Result;
+
+        static async Task<int> MainAsync()
+        {
+            (CancellationTokenSource cts, ManualResetEventSlim completed, Option<object> handler) = ShutdownHandler.Init(TimeSpan.FromSeconds(5), LoggerUtil.Writer);
+
+            // wait up to 30 seconds for debugger to attach if in a debug build
+            #if DEBUG
+            WaitForDebugger();
+#endif
+
+            LoggerUtil.Writer.LogInformation($"Starting metrics collector with the following settings:\r\n{Settings.Current}");
+            var transportSetting = new AmqpTransportSettings(TransportType.Amqp_Tcp_Only);
+            
+            ITransportSettings[] transportSettings = { transportSetting };
+            ModuleClient moduleClient = null;
+            try
+            {
+                moduleClient = await ModuleClient.CreateFromEnvironmentAsync(transportSettings);
+                moduleClient.ProductInfo = Constants.ProductInfo;
+                Option<SortedDictionary<string, string>> additionalTags = await GetAdditionalTagsFromTwin(moduleClient);
```

Comment:
@damonbarry  This was something I implemented in the test version of this module that I didn't expect to be included in the version visible to customers. I am concerned because the format in which we consume the additional tags from the twin is not polished. I forget what it is, but it is some unstructured string format (I think with semicolon delims?).

Reply:
@damonbarry Should I remove or keep it?

Reply:
My vote is that we should limit this PRs scope to just adding the monitor module to the repo. We can make this fix in another PR.

Reply:
@pmzara I agree with Andrew's latest comment, no changes necessary here. We can address the test module later.